scopes/lanes/ui/compare/lane-compare/lane-compare.tsx

```diff
@@ -11,55 +10,77 @@ import { ComponentCompareHooks } from '@teambit/component.ui.component-compare.m
 import { sortTabs } from '@teambit/component.ui.component-compare.utils.sort-tabs';
 import { LaneModel } from '@teambit/lanes.ui.models.lanes-model';
 import { LaneCompareState, computeStateKey } from '@teambit/lanes.ui.compare.lane-compare-state';
+import { LaneCompareLoader as DefaultLaneCompareLoader } from '@teambit/lanes.ui.compare.lane-compare-loader';
+import {
+  LaneCompareDrawer,
+  LaneCompareDrawerName,
+  LaneCompareDrawerProps,
+} from '@teambit/lanes.ui.compare.lane-compare-drawer';
 import { UseComponentType } from '@teambit/component';
-import { DrawerUI } from '@teambit/ui-foundation.ui.tree.drawer';
-import AnimateHeight from 'react-animate-height';
+import { H4 } from '@teambit/documenter.ui.heading';
+import {
+  useLaneDiffStatus as defaultUseLaneDiffStatus,
+  UseLaneDiffStatus,
+} from '@teambit/lanes.ui.compare.lane-compare-hooks.use-lane-diff-status';
+import { ChangeType, LaneComponentDiff } from '@teambit/lanes.entities.lane-diff';
+import classnames from 'classnames';
 
 import styles from './lane-compare.module.scss';
 
+export type LaneFilterType = ChangeType | 'ALL';
+export const ChangeTypeGroupOrder = [
+  ChangeType.NEW,
+  ChangeType.SOURCE_CODE,
+  ChangeType.ASPECTS,
+  ChangeType.DEPENDENCY,
+  ChangeType.NONE,
+];
 export type LaneCompareProps = {
   base: LaneModel;
   compare: LaneModel;
   host: string;
   tabs?: MaybeLazyLoaded<TabItem[]>;
   customUseComponent?: UseComponentType;
+  customUseLaneDiff?: UseLaneDiffStatus;
+  Drawer?: React.ComponentType<LaneCompareDrawerProps>;
+  LaneCompareLoader?: React.ComponentType;
+  ComponentCompareLoader?: React.ComponentType;
```

Comment:
can we add some comments on what each of them does? customUseLaneDiff, Drawer etc